src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -4075,8 +4079,12 @@ Map {
                   "STACKED",
                 ],
               ],
+              "isRequired": true,
```

Comment:
Is this fixing a defect? Moving a prop to be required is considered a breaking change. If it should have been this way to begin with though, we can publish it under a minor/patch version change.

Reply:
Swapped it back and set default props to type `SIMPLE`